src/prefect/environments/storage/docker.py

```diff
@@ -108,7 +102,12 @@ def __init__(
         self.files = files or {}
         self.flows = dict()  # type: Dict[str, str]
         self._flows = dict()  # type: Dict[str, "prefect.core.flow.Flow"]
-        self.base_url = base_url or default_url
+        self.base_url = environ.get(
+            "DOCKER_HOST",
+            "unix://var/run/docker.sock"
+            if sys.platform != "win32"
+            else "npipe:////./pipe/docker_engine",
+        )
```

Comment:
This doesn't allow for someone to set a `base_url` I think this should be updated to use something like
```
env var if set
else if base_url
else default_url
```

Reply:
ah yeah good point.  oversight